### ISSUE TITLE
training code

### ISSUE THREAD

**Issue by cherryolg** [ZYzhouya/Weakly-Supervised-Reinforced-Multi-operator-Image-Retargeting#1]
Hello, author. Can training codes be shared? My code ability is weak, I do not know how to realize the loss function part in the training process? I would appreciate it if you could share the training code with me!

**Comment by ZYzhouya** [comment]
> 

Please refer to this website: [http://staff.ustc.edu.cn/~chenzhibo/resources/2020/imageretargeting.html](http://staff.ustc.edu.cn/~chenzhibo/resources/2020/imageretargeting.html). Fill the form in this site, and the source code will be sent to you :)

**Comment by cherryolg** [comment]
Thank you very much for your reply. I have obtained the source code through the link above. But it only has test code and no training code, so I can't train the network myself. So can you share train.py with me? Looking forward to your reply!


------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "ZYzhouya/Weakly-Supervised-Reinforced-Multi-operator-Image-Retargeting"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2022年3月10日(星期四) 下午5:37
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [ZYzhouya/Weakly-Supervised-Reinforced-Multi-operator-Image-Retargeting] training code (Issue #1)





   
Please refer to this website: http://staff.ustc.edu.cn/~chenzhibo/resources/2020/imageretargeting.html. Fill the form in this site, and the source code will be sent to you :)
 
—
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.
